frontend/src/components/TableComponents/TableCell.tsx

```diff
@@ -1,40 +1,15 @@
-import React from 'react';
-import cn from 'classnames';
+import React, { ReactNode } from 'react';
 import styles from './cell.module.css';
-import { Button } from '../FormElements/FormElements';
 
 type TableCellProps = {
-  data: string | null;
+  data: string | ReactNode | null;
   index: number;
-  first: boolean;
-  last: boolean;
   tag?: string;
-  buttonHandler?: () => void;
-  ButtonModal?: () => JSX.Element;
```

Comment:
the button handler is needed for other tables, just not the riders table. also we need the first + last fields for the riders table or else the table cells won't be styled properly